og_migrate/src/EventSubscriber/OgMigratePreSaveSubscriber.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
These events are not fired yet. We should remove the file and update og_migrate.module's comment with some clarifying comments.

Reply:
Actually, the PRE_ROW_SAVE is fired, it is the PREPARE_ROW that is not fired.